src/test/helpers/helpers.go

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2021-Present The Zarf Authors
+
+// Package helpers provides global testing helper functions
+package helpers
+
+import (
+	"context"
+	"testing"
+)
+
+// TestContext takes a testing.T and returns a context that is
+// attached to the test by t.Cleanup()
+func TestContext(t *testing.T) context.Context {
```

Comment:
Good idea, this will be a great place to add out test logger in the future which only output logs for failed tests.

Reply:
Can I'll change pkg name to testutil